{
  try {
    this.extractor.getIntValue(null,this.person[0]);
    fail("Should have throw an exception");
  }
 catch (  final Exception e) {
  }
}
